README.md

```diff
@@ -116,15 +116,20 @@ You can run an analysis with the classic method by using one of scanners provide
 | :new:Frama-C                                          | `20.0`                                                |
 | expat                                                 | `2.0.1`                                               |
 | rats                                                  | `2.4`                                                 |
-| python                                                | `2.7.13`                                              |
-| colorama                                              | `0.3.3`                                               |
-| python-lazy-object-proxy                              | `1.2.1`                                               |
-| python-six                                            | `1.9.0`                                               |
-| wrapt                                                 | `1.10.5`                                              |
-| astroid                                               | `1.4.9`                                               |
-| Pylint                                                | `1.5`                                                 |
-| cnes-pylint-extension                                 | `1.0`                                                 |
-| python-setuptools                                     | `44.0.0`                                              |
+| python3                                               | `3.8.3`                                               |
+| pip                                                   | `20.1.1`                                              |
+| setuptools                                            | `46.1.3`                                              |
+| setuptools-scm                                        | `3.5.0`                                               |
+| pytest-runner                                         | `5.2`                                                 |
+| wrapt                                                 | `1.12.1`                                              |
+| six                                                   | `1.14.0`                                              |
+| lazy-object-proxy                                     | `1.4.3`                                               |
+| mccabe                                                | `0.6.1`                                               |
+| isort                                                 | `4.3.21`                                              |
+| typed-ast                                             | `1.4.1`                                               |
+| astroid                                               | `2.4.0`                                               |
+| pylint                                                | `2.5.0`                                               |
+| cnes-pylint-extension                                 | `5.0.0`                                               |
```

Comment:
Maybe we should add the `:new:` icon to indicate what changed.
Of course, we will need to remove this icon from tools that did not changed like SonarQube, Cppcheck, etc.

Reply:
Good idea :+1:
I'm doing it right away.